src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,36 @@
+.section-title {
+  margin: 0;
+  margin-bottom: 28px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 100%;
+  text-align: center;
+  letter-spacing: -1.5px;
+  position: relative;
+
+  @include on-tablet {
+    margin-bottom: 48px;
+    font-size: 40px;
+  }
+
+  @include on-desktop {
+    font-size: 48px;
+  }
+
+  &--align--left {
+    text-align: left;
+  }
+}
+
+.section-subtitle {
+  font-size: 16px;
+  text-align: center;
+  margin-top: -24px;
+  margin-bottom: 48px;
+  font-weight: 400;
+}
+
+.category__title {
+  color: $accent-bg-color;
+  font-size: 16px;
+}
```

Comment:
You already have a `category__title` class in the "now-on-view.scss" file; this style should be moved there